src/applications/personalization/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -45,10 +46,12 @@ class RatedDisabilityList extends React.Component {
         </>
       );
     } else {
-      headline = 'We don’t have rated disabilities on file for you';
       status = 'info';
       content = (
         <>
+          <h2 className="vads-u-margin-y--0 vads-u-font-size--lg">
```

Comment:
These headings had to be updated as well, in the event total combined rating passes, progression would be from h1 to h3 again.